### ISSUE TITLE
Screen readers support for HTML5 and WAI-ARIA

### ISSUE THREAD

**Issue by germanocorrea** [stevefaulkner/HTML5accessibility#24]
Something I think should be crucial it's information about screen readers support for HTML5 new accessibility features, including WAI-ARIA. Right now I don't know any website like HTML5accessibility with this kind of information.

Anywhay, just a tip :stuck_out_tongue:
